.codeclimate.yml

```diff
@@ -1,6 +0,0 @@
-engines:
-  hlint:
-    enabled: true
-ratings:
-  paths:
-  - "**.hs"
```

Comment:
We actually do have CodeClimate running the linter:

https://codeclimate.com/github/fission-suite/ipfs-api/issues

Reply:
hmm ok, Ill try fixing that label then :)

Reply:
nvm I see why thats returning a `?` now.